shell/src/main/java/org/apache/accumulo/shell/commands/TabletServerStatusCommand.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.shell.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.accumulo.core.client.TServerStatus;
+import org.apache.accumulo.core.client.TabletServerNotFoundException;
+import org.apache.accumulo.core.client.admin.InstanceOperations;
+import org.apache.accumulo.shell.Shell;
+import org.apache.accumulo.shell.Shell.Command;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.MissingOptionException;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+/**
+ *
+ */
+public class TabletServerStatusCommand extends Command {
+
+  private Option tserverOption, disablePaginationOpt, allOption;
+
+  @Override
+  public String description() {
+    return "get tablet servers status, master server must be running";
+  }
+
+  @Override
+  public int execute(final String fullCommand, final CommandLine cl, final Shell shellState) throws Exception {
+    List<TServerStatus> tservers;
+
+    final InstanceOperations instanceOps = shellState.getConnector().instanceOperations();
+
+    final boolean paginate = !cl.hasOption(disablePaginationOpt.getOpt());
+
+    if (cl.hasOption(tserverOption.getOpt())) {
+      tservers = new ArrayList<>();
+      for (TServerStatus ts : instanceOps.getTabletServerStatus()) {
+        if (ts.getName().equals(cl.getOptionValue(tserverOption.getOpt()))) {
+          tservers.add(ts);
+        }
+      }
+    } else if (cl.hasOption(allOption.getOpt())) {
+      tservers = instanceOps.getTabletServerStatus();
+    } else {
+      throw new MissingOptionException("Missing options");
+    }
+
+    if (tservers.isEmpty()) {
+      throw new TabletServerNotFoundException("Tablet Servers not found");
```

Comment:
I don't think we need to throw an exception here. We can just print nothing, if there's no status available.